platform/linux-generic/odp_pool.c

```diff
@@ -1123,10 +1144,14 @@ int odp_pool_capability(odp_pool_capability_t *capa)
 	capa->pkt.min_seg_len      = CONFIG_PACKET_SEG_LEN_MIN;
 	capa->pkt.max_seg_len      = max_seg_len;
 	capa->pkt.max_uarea_size   = MAX_SIZE;
+	capa->pkt.min_cache_size   = 0;
+	capa->pkt.max_cache_size   = CONFIG_POOL_CACHE_MAX_SIZE;
 
 	/* Timeout pools */
 	capa->tmo.max_pools = max_pools;
 	capa->tmo.max_num   = CONFIG_POOL_MAX_NUM;
+	capa->tmo.min_cache_size = 0;
+	capa->tmo.max_cache_size = CONFIG_POOL_CACHE_MAX_SIZE;
```

Comment:
It would probably be useful to print the cache size in `odp_pool_print()`.

Reply:
Fixed in v2